decidim-initiatives/app/controllers/decidim/initiatives/create_initiative_controller.rb

```diff
@@ -125,6 +136,14 @@ def current_initiative
         Initiative.find(session_initiative[:id]) if session_initiative.has_key?(:id)
       end
 
+      def current_organization_initiatives
+        Decidim::InitiativesType.where(organization: current_organization)
+      end
+
+      def single_initiative?
```

Comment:
Shouldn't this be ` single_initiative_type? ` or similar? Same with current_organization_initiatives -> current_organization_initiatives_types